packages/@sfcc-core/core/src/types.ts

```diff
@@ -0,0 +1,20 @@
+export type ApiConfig = {
+    config: { [key: string]: string };
+};
+export type Logger = {
+    setLevel(level: LogLevel): void;
+    log(...args: Array<string>): void;
+    info(...args: Array<string>): void;
+    debug(...args: Array<string>): void;
+    warn(...args: Array<string>): void;
+    error(...args: Array<string>): void;
+};
+export type Extension = any;
+export type Service = any;
```

Comment:
Should we add // TODO comments for future type defs? My IDE aggregates TODOs and creates a nice list for me :)

Reply:
Sure. With these though, inherently I'm not sure if there is a type-def to give them. You can register _anything_.

Reply:
Yep. agreed. More of a "discover" todo if we have a common interface type or some "pattern" we could identify as required.